Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -544,6 +544,27 @@ private DeployResult GetResult(string id, string activeDeploymentId, bool isDepl
                 string repositoryRoot = _environment.RepositoryPath;
                 var perDeploymentSettings = DeploymentSettingsManager.BuildPerDeploymentSettingsManager(repositoryRoot, _settings);
 
+                string delayMaxInStr = perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, false);
```

Comment:
the Boolean is useful when there are several setting providers, it is not very relevant in this case....
and there are code that use plain true/false as an argument:`MarkStatusComplete(currentStatus, success: false);`